content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -102,44 +167,134 @@ Adding tags to tabular metrics is similar to adding global tags, with two additi
 
     </details>
 
+{{% /collapse-content %}}
 
-### Metrics Advanced Options
 
-The advanced options for scalar and tabular metrics are the same:
+{{% /tab %}}
+{{< /tabs >}}
 
-[Metric Type][11]
-: One of `gauge`, `rate`, `monotonic_count`, or `monotonic_count_and_rate`. 
+### Step 7: Global Tags
 
-[Scale Factor][12]
-: Before being transmitted to Datadog, the extracted value is multiplied by this factor.
+Add global tags to ensure the metadata, metrics, and global tags are applied to all matching devices. Global tags can be added either from a device scan or by manually creating a new tag for the profile. Additionally, the `Inherited` tag will appear next to any global tags inherited from this profile. 
 
-[Extract Value][7]
-: This is the same as the [advanced options](#advanced-options) for global tags.
+{{< tabs >}}
+{{% tab "Device scan (recommended)" %}}
+
+1. To define a global tag using the **Device Scan** option, click **+ Add Tags**. This opens a modal displaying all available tags for the device.
+2. Select one or more tags you wish to add to the device profile, then click **Add Tag**.
+3. This returns you to the define global tags screen where you can see the new tag that was added.
```

Comment:
At this step, customers can also edit the tag to add any modification (similarly to the manual flow)
```suggestion
3. This returns you to the define global tags screen where you can see and edit the new tag that was added.
```